purchases/src/androidTestIntegrationTest/kotlin/com/revenuecat/purchases/PurchasesIntegrationTest.kt

```diff
@@ -158,6 +158,9 @@ class PurchasesIntegrationTest {
                     assertThat(offerings.current?.availablePackages?.size).isEqualTo(1)
                     assertThat(offerings.current?.monthly?.product?.sku).isEqualTo(Constants.productIdToPurchase)
 
+                    assertThat(offerings.current?.metadata).isNotNull
+                    assertThat(offerings.current?.metadata?.get("dontdeletethis")).isEqualTo("useforintegrationtesting")
```

Comment:
Just to confirm, this was added to both the load shedder project and the Main revenuecat sample project?

Reply:
Nvm! Just saw it myself 💪

Reply:
Correct correct! Tested both locally yesterday 😊 

The load shedder one "failed" yesterday but that's because of the caching but it looks like its passing today 💪